.taskcluster.yml

```diff
@@ -77,8 +77,12 @@ tasks:
           requires: all-completed   # Must be explicit because of Chain of Trust
           retries: 5
           routes:
-            - statuses  # Automatically added by taskcluster-github. It must be explicit because of Chain of Trust
-            - tc-treeherder.v2.fenix.${head_rev}
+            $flatten:
+              - statuses  # Automatically added by taskcluster-github. It must be explicit because of Chain of Trust
+              - $if: 'trust_level == 3'
+                then:
+                    - tc-treeherder.v2.fenix.${head_rev}
```

Comment:
I think that this is indented too far

Reply:
I'm sorry, I'm not sure to follow this sentence. Would you mind expanding on it?

Reply:
Sure - I think there's a formatting issue, where this line is 2 spaces too far.
Right now, it looks like:
```
          routes:
            $flatten:
              - statuses  # Automatically added by taskcluster-github. It must be explicit because of Chain of Trust
              - $if: 'trust_level == 3'
                then:
                    - tc-treeherder.v2.fenix.${head_rev}
else: []
```
and it should look like
```
          routes:
            $flatten:
              - statuses  # Automatically added by taskcluster-github. It must be explicit because of Chain of Trust
              - $if: 'trust_level == 3'
                then:
                  - tc-treeherder.v2.fenix.${head_rev}
else: []
```

Reply:
*facepalms* My apologies, I kept reading "intended too far", instead of "indented too far". Thanks for catching it, and for the thorough explanations. Fixed